test/integration/test/account.test.js

```diff
@@ -0,0 +1,9 @@
+const clientUtil = require('../src/client-util');
+const configUtil = require('../src/config-util');
```

Comment:
Minor - I would call these helpers. In my mind Utils are equivalent to simple functions that reformat strings/dates/etc